platform/admin/ui/src/main/webapp/js/models/Service.js

```diff
@@ -198,7 +198,10 @@ define(['backbone', 'jquery','backboneassociations'],function (Backbone, $) {
         initializeFromMetatype: function(metatype) {
             this.get('properties')
                 .set(metatype.reduce(function (defaults, obj) {
-                    var val = obj.get('defaultValue');
+                    var defaultVal = obj.get('defaultValue');
```

Comment:
I think the logic here would be easier to understand and maintain if we change all the ternaries into a series of if statements.  

```
var defaultVal = obj.get('defaultValue');
if (defaultVal){
     defaults[obj.get('id')] = defaultVal.toString();
} else if (obj.get('type') === 11) {
     defaults[obj.get('id')] = false;
} else {
     defaults[obj.get('id')] = null;
}
return defaults;
```